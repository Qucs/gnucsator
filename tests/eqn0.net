# Qucs 0.0.19

Eqn:Eqn1 RR0="25" Export="yes"
Eqn:Eqn2 RR1="100" Export="no"
R:R1 _net0 _net1 R="RR1" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
VProbe:Pr1 _net0 gnd
.DC:DC1
Vdc:V1 _net0 gnd U="1 V"
R:R2 _net1 gnd R="RR0"
