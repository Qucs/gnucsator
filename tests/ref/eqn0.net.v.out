
module main ();
parameter RR0={25};
parameter RR1={100};
//# Qucs 0.0.19 
Eqn #() Eqn1 ();
Eqn #() Eqn2 ();
R #(.R(RR1),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85)) R1 (.p(_net0),.n(_net1));
VProbe #() Pr1 (.p(_net0),.n(0));
Vdc #(.U(1)) V1 (.p(_net0),.n(0));
R #(.R(RR0)) R2 (.p(_net1),.n(0));
endmodule // main

